Makefile

```diff
@@ -356,43 +356,30 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-proto-all: proto-tools proto-gen proto-lint proto-check-breaking proto-swagger-gen proto-format
+proto-all: proto-format proto-lint proto-check-breaking proto-gen
 
 proto-gen:
-	@./scripts/protocgen.sh
-
-proto-gen-docker:
 	@echo "Generating Protobuf files"
 	docker run -v $(shell pwd):/workspace --workdir /workspace tendermintdev/sdk-proto-gen sh ./scripts/protocgen.sh
-.PHONY: proto-gen-docker
 
 proto-format:
 	@echo "Formatting Protobuf files"
 	docker run -v $(shell pwd):/workspace \
 	--workdir /workspace tendermintdev/docker-build-proto \
 	find ./ -not -path "./third_party/*" -name *.proto -exec clang-format -i {} \;
-.PHONY: proto-format
 
 # This generates the SDK's custom wrapper for google.protobuf.Any. It should only be run manually when needed
 proto-gen-any:
-	@./scripts/protocgen-any.sh
+	docker run -v $(shell pwd):/workspace --workdir /workspace tendermintdev/sdk-proto-gen sh ./scripts/protocgen-any.sh
```

Comment:
does this command work as intended?

Reply:
I fixed it here: https://github.com/cosmos/cosmos-sdk/pull/7931/files#diff-561f60ec6437ef1adbefc2a622282dca556dd0d07b0542be744c9db06050b38aR11

Reply:
`make` provides the automatic variable `$(CURDIR)` for that